site/docs/v1/tech/guides/multi-factor-authentication.adoc

```diff
@@ -536,8 +537,42 @@ If you have other MFA methods that you'd like to use with FusionAuth, you have a
 
 include::docs/v1/tech/guides/_mfa-migration.adoc[]
 
+== Forcing MFA
+
+There are times when you might want to force a user to MFA before they ever get access to your applciation, or force every user of a certain type to MFA.
+
+These are two different situations.
+
+=== Forcing a User to Before Accessing an Application
+
+You may want to force a user to perform MFA before they can access a certain sensitive application, and to deny them access if they have not enabled MFA.
+To do so with FusionAuth, the easiest way is to put a value in the JWT using a link:/docs/v1/tech/lambdas/jwt-populate[JWT Populate Lambda].
+
+[source,javascript]
+.Example lambda which adds MFA enabled claim
+----
+function populate(jwt, user, registration) {
+  jwt.hasMFAEnabled = user.twoFactor && user.twoFactor.methods && user.twoFactor.methods.length > 0;
+}
+----
+
+Then, when your application receives the JWT, it can check this claim.
+If the claim is missing or `false`, the application can force the user to add an MFA method or otherwise display an error.
+Such a check can happen in the application or in a proxy.
+
+Having FusionAuth https://github.com/FusionAuth/fusionauth-issues/issues/763[manage this check is an open feature request].
+Please review the issue and comment on or upvote it if you'd like to see this handled by FusionAuth.
```

Comment:
From the initial description, it sounded like this was going to tell me how to force a user to complete a second factor authentication during login, or to perform an inline step-up auth prior to accessing a secure resource. 

This description seems to be more about forcing a user to configure MFA.